src/v3/test/integration/__snapshots__/okta-verify-email-channel-enrollment.test.tsx.snap

```diff
@@ -387,7 +387,7 @@ exports[`okta-verify-email-channel-enrollment should render email channel form a
                     tabindex="0"
                     type="button"
                   >
-                    Try a different way
+                    Or &lt;a href="#" class="switch-channel-link"&gt;try a different way&lt;/a&gt; to set up Okta Verify.
```

Comment:
expected?

Reply:
Yes, this is the appropriate key for this flow, unfortunately they embed HTML in the string, I spoke with Lester on this, we will have to write some custom logic to handle this correctly, but i dont want to make the change until your changes for OV enrollment are merged because it will cause a ton of re-work. So waiting for your branch to merge first before fixing this.

Reply:
I see, maybe we can ask for a new key in translation files (followup)